src/main/java/com/password4j/SaltGenerator.java

```diff
@@ -80,4 +80,23 @@ public static byte[] generate()
         return generate(64);
     }
 
+    /**
+     * Salt by definition are shared between all passwords and
```

Comment:
Salt is not shared between passwords. Change the first phrase with "Get the length of salt from configurations."